File: Diploma OOP C#/TriangleArea.cs
using System;

namespace Area
{
    class Program
    {
        public static void Main(string[] args)
        {
            Console.WriteLine("Enter the First Number");
            double a = Convert.ToDouble(Console.ReadLine());

            Console.WriteLine("Enter the second Number");
            double b = Convert.ToDouble(Console.ReadLine());

            Console.WriteLine("Enter the Third Number");
            double c = Convert.ToDouble(Console.ReadLine());

            if ((a + b > c) && (b + c > a) && (c + a > b))
            {
                double s = (a + b + c) / 2;
                double result = Math.Sqrt(s * (s - a) * (s - b) * (s - c));
                Console.WriteLine("The result is " + result);
            }
            else
            {
                Console.WriteLine("The given sides do not form a valid triangle.");
            }

            Console.ReadKey();
        }
    }
}

File: Diploma Programming Essentials Python/TriangleArea.py
import math

a = float(input("Enter first side: "))
b = float(input("Enter second side: "))
c = float(input("Enter third side: "))

if c >= (a + b) or a >= (b + c) or b >= (c + a):
    print("Wrong input! These sides do not form a triangle.")
else:
    s = (a + b + c) / 2
    area = math.sqrt(s * (s - a) * (s - b) * (s - c))
    print(f"The area of the triangle is {area:.2f}")
